libnavigator/src/main/java/com/mapbox/navigation/navigator/LocationEx.kt

```diff
@@ -35,6 +35,7 @@ internal fun Location.toFixLocation(date: Date): FixLocation {
 
     return FixLocation(
         Point.fromLngLat(longitude, latitude),
+        elapsedRealtimeNanos,
```

Comment:
I believe we should add this to `FixLocation.toLocation()` as well

```kotlin
it.elapsedRealtimeNanos = this.monotonicTimestampNanoseconds
```

Right @kmadsen @SiarheiFedartsou @mskurydin?

Reply:
done